src/style.css

```diff
@@ -1,3 +1,81 @@
 body {
   margin: 0;
+  background-color: #e5e5e5;
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: normal;
+  text-transform: uppercase;
+  --color-link: #00acdc;
+}
+
+.header {
+  display: flex;
+  background: #fff;
+  justify-content: space-between;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  margin: 10px 50px;
+}
+
+.nav__li {
+  list-style-type: none;
+  display: inline;
+}
+
+.nav__link {
+  text-decoration: none;
+  margin-left: 17px;
+  color: #000;
+  display: inline-block;
```

Comment:
The padding value on .nav__link should be in 'em' or '%' for better responsive design